hw/ip/rv_dm/data/rv_dm_testplan.hjson

```diff
@@ -314,33 +314,18 @@
       stage: V2
       tests: []
     }
-    {
-      name: debug_req
-      desc: '''
-            Verify that the debugger can issue a debug request to the CPU.
-
-            - Set lc_hw_debug_en to non-true value and activate the dm.
-            - Set unavailable input to 0 to reflect that the hart is available.
-            - Write dmcontrol[haltreq] to 1 via JTAG and poll for dmstatus[allhalted] to be set.
-            - Verify that the debug_req output asserts.
-            - On the debug mem TL interface, after some random delay, write 1 to the `halted`
-              register in the debub memory.
-            - Ensure that the polling in the loop above completes.
-            '''
-      stage: V2
-      tests: []
-    }
     {
       name: ndmreset_req
       desc: '''
             Verify that the debugger can issue a non-debug reset to the rest of the system.
 
-            - Set lc_hw_debug_en to true value and activate the dm.
+            - Set lc_hw_debug_en / pinmux_hw_debug_en to true value and activate the dm.
             - Pick a random set of write/readable CSRs in all 3 RALs (JTAG DMI, regs RAL and debug
               mem RAL). Write known values to them.
             - Write the dmcontrol register to assert ndmreset - verify that the ndmreset output
               stays asserted.
-            - Mimic the CPU going into reset by asserting unavailable input.
+            - Mimic the CPU going into reset by asserting unavailable input and setting
```

Comment:
In full chip, the core domain reset will also assert. We should model that also here, right?

Reply:
Good point. I have amended this testpoint to include `rst_ni` as well.